third_party/nvfuser/csrc/scheduler/reduction_heuristic.h

```diff
@@ -110,6 +114,16 @@ class ReductionParams : public HeuristicParams {
         unroll_factor_outer_reduction > 1;
   }
 
+  // specific to combined inner and outer reduction?
+  bool combined_inner_outer = false;
+  bool tidx_for_outer_reduction = false;
+  bool pad_outer_reduction_to_warp = false;
+  int64_t vectorization_factor_tmp_gmem_read = 1;
+  int64_t vectorization_factor_tmp_gmem_write = 1;
+  // for combined reduciton, inner reduciton domain is paralled by both TIDx and
```

Comment:
typo: parallelized

Reply:
The comment isn't clear what the parameter parallel type means.

Reply:
added. `  // inner reduciton axis is paralled by block_dim_inner_reduction (usually TIDx)
  // the remaining part is further paralled by block_dim_inner_reduction_extra (usually TIDy)`